### PR TITLE
Move css, modules, chart_class to 3rd parameter

### PR DESCRIPTION
**Pull request by Prakriti-nith**
Shifted css, modules, chart_class to the 3rd parameter in highcharts examples.

### CODE REVIEW

**Comment by Prakriti-nith** [comment]
@Shekharrajak ping!

File: demo_rails/app/controllers/highchart_stock_map_controller.rb [hunk 1]
```diff
@@ -94,6 +93,10 @@ def highchart_stock_map
       }
     }
 
+    user_opts2 = {
```
Comment (Shekharrajak) [inline demo_rails/app/controllers/highchart_stock_map_controller.rb]: Very bad variable name.

File: demo_rails/app/controllers/highchart_stock_map_controller.rb [hunk 2]
```diff
@@ -59,13 +59,12 @@ def highchart_stock_map
       }])
 
     # initialize
-    @stock = Daru::View::Plot.new(series_dt, opts)
+    @stock = Daru::View::Plot.new(series_dt, opts, user_opts)
 
     @hchart = Daru::View::Plot.new(series_dt, opts3)
 
 
     opts2 = {
```
Comment (Shekharrajak) [inline demo_rails/app/controllers/highchart_stock_map_controller.rb]: change the name.

File: demo_sinatra/app.rb [hunk 3]
```diff
@@ -554,7 +554,7 @@ def highchart_stock_map
     }
   ]
 
-  @area_range = Daru::View::Plot.new(series_dt6, opts6)
+  @area_range = Daru::View::Plot.new(series_dt6, opts6, user_options)
 
   opts8 = {
```
Comment (Shekharrajak) [inline demo_sinatra/app.rb]: Very disappointed to see variable names  :(

**Review by Shekharrajak — CHANGES_REQUESTED** [review]

File: demo_sinatra/app.rb (hunk 3 above)
Comment (Prakriti-nith) [reply demo_sinatra/app.rb]: Really sorry about this. As discussed, I just added the examples that I was using locally to check the features. I will fix this soon.

**Comment by Shekharrajak** [comment]
@Prakriti-nith , why it is still open?

**Comment by Prakriti-nith** [comment]
@Shekharrajak it is complete. I forgot to comment after the commit. It can be merged.

**Review by Shekharrajak — APPROVED** [review]

**Pull request merged**